tests/model/test_user.py

```diff
@@ -0,0 +1,29 @@
+from tornado.testing import gen_test, AsyncTestCase
+
+from bootcamp.models.user import User
+
+
+class TestUser(AsyncTestCase):
+    @gen_test
+    def test_get(self):
+        uuid = '1cf41b3d-f7ad-4238-bf08-8794cf7ae0f4'
+        obj = User.get(uuid)
+        assert isinstance(obj, User) == False
```

Comment:
Should be `assert isinstance(obj, User) is False` here, my fault.
I'll change it in other places.